explorer/composables/useEra5WrfSeries.ts

```diff
@@ -0,0 +1,59 @@
+import type { Ref } from 'vue'
```

Comment:
I think this composable possibly makes sense for coordinating state between the individual charts however there's other vue-y ways this happens and it won't be reused outside of this xray probably.  Discuss.